DBAL/TracingStatement.php

```diff
@@ -16,9 +16,9 @@ final class TracingStatement implements IteratorAggregate, StatementCombinedResu
      * @var Statement<Statement>
      */
     private $statement;
-    private $sql;
-    private $spanFactory;
-    private $username;
+    private string $sql;
+    private SpanFactory $spanFactory;
+    private ?string $username = null;
```

Comment:
Same as above. Do we really need to default it with null?

Reply:
No, probably not, I will change that.